engine/admin/index.md

```diff
@@ -159,7 +159,7 @@ you installed Docker, you may already have this file.
 7. Verify that the `docker` daemon is running as specified with the `ps` command.
 
     ```bash
-    $ ps aux | grep docker | grep -v grep
+    $ ps aux | grep [d]ocker
```

Comment:
It's a little trick that's hard to explain, but the `grep` result from `ps` will contain the bracket so it won't match the regular expression. Try it and see.